Makefile

```diff
@@ -172,3 +172,9 @@ get-cluster-credentials: set-azure-account
 bin/konduit.sh:
 	curl -s https://raw.githubusercontent.com/DFE-Digital/teacher-services-cloud/main/scripts/konduit.sh -o bin/konduit.sh \
 		&& chmod +x bin/konduit.sh
+.PHONY: install-konduit
```

Comment:
this is already in the Makefile as bin/konduit.sh at line 172

Reply:
used existing one from make file